server/api/src/rsDataObjChksum.cpp

```diff
@@ -315,13 +491,12 @@ dataObjChksumAndRegInfo( rsComm_t *rsComm, dataObjInfo_t *dataObjInfo,
     return status;
 }
 
-int
-verifyDatObjChksum( rsComm_t *rsComm, dataObjInfo_t *dataObjInfo,
-                    char **outChksumStr ) {
-    int status;
-
+int verifyDatObjChksum(rsComm_t* rsComm,
```

Comment:
Consider marking `verifyDatObjChksum` as deprecated, and have it and everywhere else call a new, correctly spelled `verifyDataObjChksum` with an `a`.